Stresser/RegistryArtifact.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include "IArtifact.h"
+#include "../Utils/AutoRegistryKeyHandle.h"
+#include "../Utils/RegistryArtifactUtils.h"
+
+class RegistryArtifact : public IArtifact
+{
+public:
+	explicit RegistryArtifact(std::string name, std::string data);
+	virtual ~RegistryArtifact();
+
+	virtual void install() const;
+	virtual void uninstall() const;
+
+private:
+	HKEY registryPrefix;
```

Comment:
Suggestion: No need for SmartHandle? if it is the case, maybe add a comment ?

Reply:
the HKEY is const like HKLM and i dont think you need smartHnadle for this

Reply:
👍